autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>adhere to the indentation style as in the sdk and elsewhere?</p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; automl = AutoML(
        role=role,
&nbsp; &nbsp; &nbsp; &nbsp; target_attribute_name=target_attribute_name,
&nbsp; &nbsp; &nbsp; &nbsp; base_job_name=base_job_name,
&nbsp; &nbsp; &nbsp; &nbsp; sagemaker_session=session,
&nbsp; &nbsp; &nbsp; &nbsp; problem_type='BinaryClassification',
&nbsp; &nbsp; &nbsp; &nbsp; job_objective={'MetricName': eval_obj},
&nbsp; &nbsp; &nbsp; &nbsp; max_candidates=250
&nbsp; &nbsp; ) 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
done